src/backend/joanie/core/models.py

```diff
@@ -1 +1,104 @@
 # Create your models here.
+
+from django.db import models
+
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from parler import models as parler_models
+
+
+#from . import enums
+
+# move to enums.py
+PRODUCT_TYPE_CHOICES = (
+    ('credential', _("Credential")),
+    ('enrollment', _("Enrollment")),
+    ('certificate', _("Certificate")),
+)
+
+ORDER_STATUS_CHOICES = (
+    ('pending', _("Pending")),
+    ('in_progress', _("In progress")),
+    ('finished', _("Finished")),
+)
+
+ENROLLMENT_STATUS_CHOICES = (
+    ('in_progress', _("In progress")),
+    ('validated', _("Validated")),
+    ('failed', _("Failed")),
+)
+
+
+class Course(models.Model):
+    """ A new course created will initialize a cms page """
+    code = models.CharField(verbose_name=_("reference to cms page"), blank=True, max_length=100)
+    title = models.CharField(verbose_name=_("title"), max_length=255) # will be used to generate a slug name of cms page
+
+
+class Certification(models.Model):
+    """ Certificate definition """
+    name = models.SlugField(verbose_name=_("name"), max_length=255, unique=True)
+    title = models.CharField(verbose_name=_("title"), max_length=255)
+    description = models.TextField(verbose_name=_("description"), max_length=500)
+    # howard template used to generate pdf certificate
+    template = models.CharField(verbose_name=_("template to generate pdf"))  # ??
+
+
+class Product(models.Model):
+    type = models.CharField(verbose_name=_("type"), choices=PRODUCT_TYPE_CHOICES, max_length=50)
+    name = models.SlugField(verbose_name=_("name"), max_length=255, unique=True)
+    title = models.CharField(verbose_name=_("title"), max_length=255)
+    price = models.CharField(verbose_name=_("price"), blank=True, max_length=100)
+    course = models.ForeignKey(Course, verbose_name=_("course"))  # link between cms page and product
```

Comment:
I'm rethinking this...
because this would force us to create a product for each course with its course runs below...

Instead we could have just a few products and the course runs would point to a course and to a product.
It achieves the same but products are defined only once... and for each course the course runs are linked via a model pointing to three models:
```python
class ProductDefinition
    course
    course_run
    product
```